packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -36,12 +36,12 @@ const MenuComp = React.forwardRef<HTMLDivElement, MenuProps>(
         {...otherProps}
         aria-label={ariaLabel}
         backgroundColor="elevated"
-        borderRadius="default"
+        borderRadius={styles.container.borderRadius}
         boxShadow="z3"
         display="inline-block"
         onKeyDown={handleKeys}
-        paddingX={0.5}
-        paddingY={0.5}
+        paddingX={styles.container.padding}
+        paddingY={styles.container.padding}
```

Comment:
i know this was already like this, but it can just be `padding={styles.container.padding}`